ostd/src/mm/stat/allocated.rs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Allocated memory
+
+use core::sync::atomic::{AtomicUsize, Ordering};
+
+static ALLOCATED: AtomicUsize = AtomicUsize::new(0);
```

Comment:
I suggest maintaining the allocated and total states directly in the page::allocator module and read them in `ostd::mm::stat`. There's no need to be two individual modules called `stat::allocated` and `stat::total`.